backend/bridgecommon/bridgecommon.go

```diff
@@ -363,8 +363,9 @@ func Shutdown() {
 
 	log := logging.Get().WithGroup("server")
 	if globalShutdown != nil {
+		log.Info("Shutdown about to be called")
```

Comment:
nit: is this still needed, or is it a debug leftover?

Reply:
I thought it's a helpful line anyway in case Shutdown gets stuck for whatever reason